src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala

```diff
@@ -29,16 +29,16 @@ object DocumentationGenerator extends App {
 
   // This impacts what gets displayed on the main documentation index.
   val stableVersions = Seq(
-    VersionSummary("1.2.x", s"Lagom $currentLagomVersion (current stable release) for Scala 2.11"),
-    VersionSummary("1.1.x", s"Lagom 1.1.0 (previous stable release) for Scala 2.11")
+    VersionSummary("1.2.x", s"Lagom $currentLagomVersion (current stable release)"),
+    VersionSummary("1.1.x", s"Lagom 1.1.0 (previous stable release)")
   )
 
   val previewVersions = Seq(
     VersionSummary("1.3.x", s"Lagom 1.3.0-M1 for Scala 2.11")
```

Comment:
I wonder whether we even want it here. I feel like it would be more natural to put it in the "Scala" sub-heading.